app/controllers/payments_controller.rb

```diff
@@ -55,7 +55,7 @@ def validate_request
       message = "Failed to recognize payment order: #{order}, status: #{status}"
       Raven.capture_exception(message)
       # Don't give anything away to attackers poking at the system
-      render nothing: true
```

Comment:
render nothing: true  or render :nothing is going in rails 5.1 - so now we are ready